docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>Do we have an issue in Cirq for crafting a Sampler-like API which can run calibrations so we don't have to deal with "bare" engine objects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I just created https://github.com/quantumlib/Cirq/issues/3788</p>